client/dive-common/components/RunTrainingMenu.vue

```diff
@@ -130,10 +130,11 @@ export default defineComponent({
 
         <v-alert
           dense
-          type="warning"
+          color="warning"
+          class="mx-2"
```

Comment:
Feel entirely free to ignore:  `outlined`? 
![image](https://user-images.githubusercontent.com/61746913/111472877-cb919e80-8700-11eb-87bd-745c7ebda560.png)
![image](https://user-images.githubusercontent.com/61746913/111472944-dfd59b80-8700-11eb-8715-7cd20a44ec49.png)